core/gui/src/app/workspace/component/result-panel/result-panel.component.ts

```diff
@@ -283,12 +286,33 @@ export class ResultPanelComponent implements OnInit, OnDestroy {
     this.dragPosition = { x: this.returnPosition.x, y: this.returnPosition.y };
   }
 
+  isPanelDocked() {
+    return this.returnPosition.x === this.dragPosition.x && this.returnPosition.y === this.dragPosition.y;
+  }
+
+  handleEndDrag({ source }: CdkDragEnd) {
+    const { x, y } = source.getFreeDragPosition();
+    this.dragPosition = { x: x, y: y };
```

Comment:
a short comment on this function to explain that this function is used to record the up-to-date panel position

Reply:
added comment